### DISCUSSION TITLE
Email Login Not Working On Mainnet

### DISCUSSION THREAD

**Discussion by metasoul111** [Web3Auth/Web3Auth#1279] [Help]
Once you receive the email verification email and click on the button this is the error you get.

No mail generated. Please initiate the passwordless flow first.

This is happening even when trying to log in to https://dashboard.web3auth.io/

![Screenshot from 2023-02-03 17-01-36](https://user-images.githubusercontent.com/89988259/216727091-b098b252-1d04-43fc-8397-2f598598a565.png)

**Comment by metasoul111** [comment]
@chaitanyapotti

**Comment by YZhenY** [comment]
Some times your session gets stale when theres a verision upgrade, please do a hard refresh and things shoudl work

**Comment by metasoul111** [comment]
@YZhenY

Did that didn't work, here are console logs as requested.


![Screenshot from 2023-02-03 20-38-49](https://user-images.githubusercontent.com/89988259/216742185-a4c01235-d41c-44f7-8abc-83c87947096c.png)

**Reply by tanguyenvn** [reply]
Can you give an update if the error still happens to you?

**Reply by metasoul111** [reply]
@tainguyentt issue is fixed

**Reply by jdnichollsc** [reply]
I'm getting this error too

![Screenshot_20230207-135049_Chrome.jpg](https://user-images.githubusercontent.com/2154886/217339478-ff27f19a-a894-4dca-b5b0-2789866d2ff9.jpg)

Any help is really appreciated!

**Reply by YZhenY** [reply]
any console logs and region you're from please?

**Reply by jdnichollsc** [reply]
@YZhenY I was testing this from a mobile device in Colombia, I don't have logs but let me try to reproduce this from desktop too

**Reply by aaronwilly** [reply]
This is still not working.
